src/20241108_Multi_OnboardAndEnableSUSDeLiquidEModeOnAaveV3MainnetAndLidoInstances/AaveV3EthereumLido_OnboardAndEnableSUSDeLiquidEModeOnAaveV3MainnetAndLidoInstances_20241108.sol

```diff
@@ -0,0 +1,120 @@
+
+  address public constant sUSDe = 0x9D39A5DE30e57443BfF2A8307A4256c8797A3497;
+  uint256 public constant sUSDe_SEED_AMOUNT = 100 * 1e18;
+  address public constant sUSDe_ADMIN = 0xac140648435d03f784879cd789130F22Ef588Fcd;
+
+  function _postExecute() internal override {
+    IERC20(sUSDe).forceApprove(address(AaveV3EthereumLido.POOL), sUSDe_SEED_AMOUNT);
+    AaveV3EthereumLido.POOL.supply(
+      sUSDe,
+      sUSDe_SEED_AMOUNT,
+      address(AaveV3EthereumLido.COLLECTOR),
+      0
+    );
+
+    (address asUSDe, , ) = AaveV3EthereumLido.AAVE_PROTOCOL_DATA_PROVIDER.getReserveTokensAddresses(
+      sUSDe
+    );
+    IEmissionManager(AaveV3EthereumLido.EMISSION_MANAGER).setEmissionAdmin(sUSDe, sUSDe_ADMIN);
+    IEmissionManager(AaveV3EthereumLido.EMISSION_MANAGER).setEmissionAdmin(asUSDe, sUSDe_ADMIN);
+  }
+
+  function eModeCategoriesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.EModeCategoryUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.EModeCategoryUpdate[]
+      memory eModeUpdates = new IAaveV3ConfigEngine.EModeCategoryUpdate[](1);
+
+    eModeUpdates[0] = IAaveV3ConfigEngine.EModeCategoryUpdate({
+      eModeCategory: 2,
+      ltv: 90_00,
+      liqThreshold: 92_00,
+      liqBonus: 3_00,
+      label: 'sUSDe'
+    });
+
+    return eModeUpdates;
+  }
+  function assetsEModeUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.AssetEModeUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.AssetEModeUpdate[]
+      memory assetEModeUpdates = new IAaveV3ConfigEngine.AssetEModeUpdate[](3);
+
+    assetEModeUpdates[0] = IAaveV3ConfigEngine.AssetEModeUpdate({
+      asset: sUSDe,
+      eModeCategory: 2,
+      borrowable: EngineFlags.DISABLED,
+      collateral: EngineFlags.ENABLED
+    });
+    assetEModeUpdates[1] = IAaveV3ConfigEngine.AssetEModeUpdate({
+      asset: AaveV3EthereumLidoAssets.USDS_UNDERLYING,
+      eModeCategory: 2,
+      borrowable: EngineFlags.ENABLED,
+      collateral: EngineFlags.DISABLED
+    });
+    assetEModeUpdates[2] = IAaveV3ConfigEngine.AssetEModeUpdate({
+      asset: AaveV3EthereumLidoAssets.USDC_UNDERLYING,
+      eModeCategory: 2,
+      borrowable: EngineFlags.ENABLED,
+      collateral: EngineFlags.DISABLED
+    });
+
+    return assetEModeUpdates;
+  }
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: sUSDe,
+      assetSymbol: 'sUSDe',
+      priceFeed: 0xb37aE8aBa6C0C1Bf2c509fc06E11aa4AF29B665A,
+      enabledToBorrow: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 5,
+      liqThreshold: 10,
+      liqBonus: 7_50,
+      reserveFactor: 10_00,
+      supplyCap: 20_000_000,
+      borrowCap: 0,
```

Comment:
Same as previous, better to put borrowCap of some low value like 1000 as borrowCap of 0 means no cap at all.